kythe/cxx/indexer/textproto/analyzer.cc

```diff
@@ -95,6 +96,13 @@ proto::VName LookupVNameForFullPath(absl::string_view full_path,
   return proto::VName{};
 }
 
+// The TreeInfo contains the ParseInfoTree from proto2 textformat parser
+// and line offset of the textproto within a file.
+struct TreeInfo {
+  const TextFormat::ParseInfoTree& parse_tree;
+  int line_offset;
```

Comment:
I didn't make it member, since ParseInfoTree was being passed around and also, passing this offset along makes it easy to do threading when there is multiple chunk with big recordio files.